app/scripts/modules/azure/loadBalancer/configure/createLoadBalancer.controller.spec.js

```diff
@@ -20,51 +22,37 @@ describe('Controller: azureCreateLoadBalancerCtrl', function () {
       isNew: true
     });
   }));
-
-  it('requires health check path for HTTP/S', function () {
-    var loadBalancer = {
-      healthCheckProtocol: 'HTTP'
-    };
-
-    this.$scope.loadBalancer = loadBalancer;
-
-    expect(this.ctrl.requiresHealthCheckPath()).toBe(true);
-
-    loadBalancer.healthCheckProtocol = 'HTTPS';
-    expect(this.ctrl.requiresHealthCheckPath()).toBe(true);
-
-    loadBalancer.healthCheckProtocol = 'SSL';
-    expect(this.ctrl.requiresHealthCheckPath()).toBe(false);
-
-    loadBalancer.healthCheckProtocol = 'TCP';
-    expect(this.ctrl.requiresHealthCheckPath()).toBe(false);
-
+  
+  beforeEach(window.inject(function($httpBackend) {
+     // Set up the mock http service responses
+     $http = $httpBackend;
+   }));
+
+  it('correctly creates a default loadbalancer', function() {
+    var lb = this.$scope.loadBalancer;
+    
+    expect(lb.probes.length).toEqual(1);
+    expect(lb.loadBalancingRules.length).toEqual(1);
+    
+    expect(lb.loadBalancingRules[0].protocol).toEqual('TCP');
+    
+    expect(this.$scope.existingLoadBalancerNames).toEqual(undefined);
+    expect(lb.ProviderType).toEqual(undefined);
```

Comment:
What's the intent of this line? Should it be testing against `providerType`, maybe?